packages/core/src/stack-layout/StackLayout.tsx

```diff
@@ -10,14 +25,18 @@ export type StackLayoutProps<T extends ElementType> =
        * Defines the default behavior for how flex items are laid out along the cross axis on the current line, default is "stretch".
        */
       align?: FlexLayoutProps<ElementType>["align"];
+      /**
+       * Establishes the main-axis, defining the direction children are placed. Default is "column".
+       */
+      direction?: ResponsiveProp<LayoutDirection>;
       /**
        * Controls the space between items, default is 3.
        */
       gap?: FlexLayoutProps<ElementType>["gap"];
       /**
        * Adds a separator between elements, default is false.
        */
-      separators?: FlexLayoutProps<ElementType>["separators"];
+      separators?: LayoutSeparator | true;
```

Comment:
Do you think seperators should also allow false? E.g. `<StackLayout seperators={false} />` seems valid e.g. if you want to conditionally toggle them on or off